content/en/docs/concepts/policy/node-resource-managers.md

```diff
@@ -182,27 +185,38 @@ equal to one. The `nginx` container is granted 2 exclusive CPUs.
 
 #### Static policy options {#cpu-policy-static--options}
 
-The behavior of the static policy can be fine-tuned using the CPU Manager policy options.
-The following policy options exist for the static CPU management policy:
-{{/* options in alphabetical order */}}
```

Comment:
This `{{/*  options */ }}` variable [doesn't work on the current website page](https://kubernetes.io/docs/concepts/policy/node-resource-managers/#cpu-policy-static--options).
A review is required to my changes below.